.github/workflows/ci.yaml

```
@@ -0,0 +1,34 @@
+name: rosinstall-generator-ci
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: ['*']
+
+jobs:
+    build:
+      strategy:
+        matrix:
+          os: [ubuntu-latest, macos-latest]
+          python: [3.7, 3.8, 3.9]
+          include:
+          - os: ubuntu-18.04
+            python: 2.7
+          - os: ubuntu-18.04
+            python: 3.6
+      name: rosinstall_generator tests
+      runs-on: ${{matrix.os}}
+
+      steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{matrix.python}}
+        uses: actions/setup-python@v1
```

Any reason not to use v2 of this action?
```suggestion
        uses: actions/setup-python@v2
```

---

Nope, just making minimal changes to stale PRs and didn't notice.